src/components/ErrorMessage/ErrorMessage.jsx

```diff
@@ -0,0 +1,12 @@
+import PropTypes from "prop-types";
+import "./ErrorMessage.css";
+
+const ErrorMessage = ({ error }) => {
+  return <p className="error-message">{error}</p>;
```

Comment:
What happens if you pass an empty string, the `p` tag would appear anyway, so it's better to do something like:

```suggestion
  return error ? <p className="error-message">{error}</p> : null;
```